### DISCUSSION TITLE
new subcommand: `bashly new`

### DISCUSSION THREAD

**Discussion by meleu** [bashly-framework/bashly#584] [Ideas]
It would be nice to have a `bashly new` subcommand, kinda similar to `rails`.

So, if I run:
```
bashly new MY_CLI_APP
```

It would generate a new directory named `MY_CLI_APP` and fill the `src/bashly.yml` file with `name: MY_CLI_APP` instead of `name: cli`.

**Comment by DannyBen** [comment]
Well, I was never a big fan of `rails new` - if you need a generator to generate the generator config, then there is a problem.

Having both `bashly new` and `bashly init` seems confusing.
So what is left is adding an argument to `bashly init` so that it will influence the name of the CLI, which I don't think is needed.

Since you must edit the YAML anyway after you create it with `bashly init`, I see no point in *just* providing an argument that changes one option (`name=your_name`) in it.

Am I missing something?

**Reply by meleu** [reply]
> Having both `bashly new` and `bashly init` seems confusing.

You're right. The features I initially envisaged for quickly starting a new project would mostly go into `bashly init` and the `new` would be just a wrapper around it creating a new directory.

I'm gonna close this one.
